test/test_mps.py

```diff
@@ -8877,8 +8877,8 @@ def test_cpu_indices(self, device="mps"):
 class TestRNNMPS(TestCaseMPS):
     def test_lstm_1(self, device="mps", dtype=torch.float32):
 
-        rnn = nn.LSTM(1, 4, 2, device="cpu")
-        input = torch.randn(2, 3, 1, device="cpu")
+        rnn = nn.LSTM(7, 4, 2, device="cpu")
+        input = torch.randn(2, 3, 7, device="cpu")
         hx = torch.zeros(2, 3, 4, device="cpu")
         cx = torch.zeros(2, 3, 4, device="cpu")
```

Comment:
This fails if initialized without zeros!!

LSTM forward pass with multiple layers is currently broken!

Reply:
Hope to find the bug in the next commits. It passes now as, clearly, when all states are zeroed, they do not make the difference